.github/workflows/bridge.yml

```diff
@@ -0,0 +1,158 @@
+name: tardis-carsus-compatibility-check
+
+on: 
+  workflow_dispatch:
+    inputs:
+      git-ref:
+        description: Git Ref (Optional)    
+        required: false
+        
+env:
+  XUVTOP: /tmp/chianti
+  CHIANTI_DL_URL: https://download.chiantidatabase.org
+  CHIANTI_DB_VER: CHIANTI_v9.0.1_database.tar.gz
+  NBCONVERT_CMD: jupyter nbconvert --execute --ExecutePreprocessor.timeout=600 --to html
```

Comment:
put the flags into a variable instead of the full command

Reply:
Done in https://github.com/tardis-sn/carsus/pull/313/commits/ecbf35cbfeac29cbf45ff795d851e87c690c93d5